packages/core/src/Utils/Events/storage.js

```diff
@@ -10,3 +10,13 @@ export const setStorageEvents = () => {
         }
     });
 };
+
+export const getAllowedLocalStorageOrigin = () => {
+    if (/^staging\.deriv\.app$/i.test(window.location.hostname)) {
+        return 'https://smarttrader-staging.deriv.app';
+    } else if (/^deriv\.app$/i.test(window.location.hostname)) {
+        return 'https://smarttrader.deriv.app';
+    }
+    // TODO: On merge, this should default to 'https://smarttrader.deriv.app'.
+    return 'https://aaron-binary3.binary.sx';
```

Comment:
Same here, I remember I left a comment on Gitlab to prevent accidental merge, so doing the same here.